iptables/builder/builder.go

```diff
@@ -57,7 +57,7 @@ func (t *IPTables) Build(verbose bool) string {
 	return strings.Join(tables, separator) + "\n"
 }
 
-func BuildIPTables(cfg config.Config) (string, error) {
+func BuildIPTables(cfg config.Config, ipv6 bool) (string, error) {
```

Comment:
What's the difference between the `ipv6` flag passed around some of these functions vs the `cfg.ipv6`? All the functions have the `cfg` object available, can't they divine the ipv6 setting from that without the additional flag?